entities/entities/spawned_money/cl_init.lua

```diff
@@ -3,27 +3,25 @@ include("shared.lua")
 function ENT:Initialize()
 end
 
+local vector_up = Vector(0,0,1)
 function ENT:Draw()
     self:DrawModel()
+    local offset = (self:OBBMaxs() - self:OBBCenter())*vector_up
```

Comment:
Also, OBBMaxs - OBBCenter shouldn't get you to the top middle of an object, which is probably what you'd want. With the standard money packet, it just gets the Pos (which is either bottom middle or center middle) and adds Ang:Up() * 0.9 to it to go to the top middle of the money packet.

OBBMaxs - OBBCenter is the local vector going from the center to the top right. Surely that shouldn't go straight up for most models.